sdk/resources/mgmt/src/main/java/com/azure/resourcemanager/resources/fluentcore/model/implementation/AcceptedImpl.java

```diff
@@ -0,0 +1,161 @@
+    private SyncPoller<Void, T> syncPoller;
+
+    public AcceptedImpl(Response<Flux<ByteBuffer>> activationResponse,
+                        SerializerAdapter serializerAdapter,
+                        HttpPipeline httpPipeline,
+                        Type pollResultType,
+                        Type finalResultType,
+                        Function<InnerT, T> wrapOperation) {
+        this.activationResponse = Objects.requireNonNull(activationResponse);
+        this.serializerAdapter = Objects.requireNonNull(serializerAdapter);
+        this.httpPipeline = Objects.requireNonNull(httpPipeline);
+        this.pollResultType = Objects.requireNonNull(pollResultType);
+        this.finalResultType = Objects.requireNonNull(finalResultType);
+        this.wrapOperation = Objects.requireNonNull(wrapOperation);
+    }
+
+    @Override
+    public T getAcceptedResult() {
+        try {
+            return wrapOperation.apply(serializerAdapter.deserialize(
+                new String(getResponse(), StandardCharsets.UTF_8),
+                finalResultType,
+                SerializerEncoding.JSON));
+        } catch (IOException e) {
+            return null;
+        }
+    }
+
+    @Override
+    public SyncPoller<Void, T> getSyncPoller() {
+        if (syncPoller == null) {
+            syncPoller = new SyncPollerImpl<InnerT, T>(this.getPollerFlux().getSyncPoller(), wrapOperation);
+        }
+        return syncPoller;
+    }
+
+    private PollerFlux<PollResult<InnerT>, InnerT> getPollerFlux() {
+        if (pollerFlux == null) {
+            Flux<ByteBuffer> content = Flux.just(ByteBuffer.wrap(getResponse()));
+            Response<Flux<ByteBuffer>> clonedResponse = new SimpleResponse<>(activationResponse, content);
+
+            pollerFlux = PollerFactory.create(
+                serializerAdapter,
+                httpPipeline,
+                pollResultType,
+                finalResultType,
+                SdkContext.getLroRetryDuration(),
+                Mono.just(clonedResponse)
+            );
+        }
+        return pollerFlux;
+    }
+
+    @Override
+    public T getFinalResult() {
+        return this.getSyncPoller().getFinalResult();
+    }
+
+    private byte[] getResponse() {
+        if (responseBytes == null) {
+            responseBytes = FluxUtil.collectBytesInByteBufferStream(activationResponse.getValue()).block();
+        }
+        return responseBytes;
+    }
+
+    private static class SyncPollerImpl<InnerT, T>
+        implements SyncPoller<Void, T> {
+
+        private final SyncPoller<PollResult<InnerT>, InnerT> syncPoller;
+        private final Function<InnerT, T> wrapOperation;
+        private T finalResult;
+
+        SyncPollerImpl(SyncPoller<PollResult<InnerT>, InnerT> syncPoller, Function<InnerT, T> wrapOperation) {
+            this.syncPoller = syncPoller;
+            this.wrapOperation = wrapOperation;
+        }
+
+        @Override
+        public PollResponse<Void> poll() {
```

Comment:
Next plan is to give a concrete class for this current `Void`, to provide more information, e.g. poll url, response body, etc.

But that would require more changes in core-management, so it would likely be delayed to another PR.